nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -742,6 +743,40 @@ in
         '';
       };
 
+    virtualisation.directBoot = {
+      enable =
+        mkOption {
+          type = types.bool;
+          default = !cfg.useBootLoader;
+          defaultText = "!cfg.useBootLoader";
+          description =
+            lib.mdDoc ''
+              If enabled, the virtual machine will direct boot the system
+              by passing the kernel, initrd and relevant parameters to QEMU.
```

Comment:
```suggestion
              If enabled, the virtual machine will boot directly into the kernel instead of through a bootloader. Other relevant parameters such as the initrd are also passed to QEMU.
```